### PR TITLE
2705 3   career videos   update administrate on curriculum

### PR DESCRIPTION
**Pull request by A-Wheeto**
## Status

* Current Status: Ready for (front-end / tech) review
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2705
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2706  
## Review progress:

- [ ] Browser tested
- [ ] Front-end review completed
- [ ] Tech review completed

## What's changed?

- Created a new administrate dashboard for videos
- Added video option onto units and lessons administrate dashboards
- Added new graphql definitions
- Added tests for new dashboard and graphql

### CODE REVIEW

File: spec/factories/video.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :video do
+    title { "MyVideo" }
+    video_url { "www.example.com" }
```
Comment (msquance-stem) [inline spec/factories/video.rb:4]: Similar to the other branch, can we make this a youtube url?

**Review by msquance-stem — COMMENTED** [review]
Just one comment on the testing file

File: spec/factories/video.rb (hunk 1 above)
Comment (A-Wheeto) [reply spec/factories/video.rb:4]: Agreed. I will update this one.

**Review by msquance-stem — APPROVED** [review]
LGTM

**Pull request merged**